vumi/transports/smpp/smpp_transport.py

```diff
@@ -299,6 +299,16 @@ def _reject_for_invalid_address(self, message, field):
             message['message_id'], u'Invalid %s: %s' % (field, message[field]))
 
     @inlineCallbacks
+    def on_smpp_bind(self):
+        yield self.publish_status_bound()
+
+    def publish_status_bound(self):
+        return self.publish_status('smpp', 'good', reasons=[{
+            'type': 'bound',
+            'message': 'Bound',
+        }])
```

Comment:
I don't think we have any information at this point that would be useful? I think things like the time it took to bind, or smpp version. But I don't think that any of those things are easily available here, we would have to implement them separately. I think for now we can leave details empty, and if we find that some things are useful to know, then we can implement them.